examples/custom-tolerations/main.tf

```diff
@@ -0,0 +1,21 @@
+provider "kubernetes" {}
+
+module "lacework_k8s_datacollector" {
+  source = "../../"
```

Comment:
Can we have an example where Terraform pull this modules from GitHub? It would help users that are not very familiar with Terraform.